### PR TITLE
January project update

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
- Jquery Version Bump
- Adds new Project Update page & updates nav
- Fixes highcharts issue that was breaking formatting in test environment (this may have been exclusive to my test environment but was the blocker for opening this PR)

## How to test
All changes here are also live in the test environment at the following link: https://reitermb.github.io/Test/

### CODE REVIEW

**Review by lfrohlich — APPROVED** [review]
Approved pending a11y review

**Comment by ttran-hub** [comment]
a11y looks good 🚀

**Comment by andrew-jameson** [comment]
If this is good to go, can the "ready to merge" label be added so I know to merge? Seems like all review is done.

cc @reitermb

**Comment by ADPennington** [comment]
> If this is good to go, can the "ready to merge" label be added so I know to merge? Seems like all review is done.
> 
> cc @reitermb

I included some feedback [here](https://github.com/raft-tech/TANF-app/issues/2298#issuecomment-1379393015). goal is for Miles to update before end of week and tag OFA team for final review. cc: @ttran-hub @lfrohlich

File: product-updates/index.html [hunk 1]
```diff
@@ -128,78 +237,366 @@
...
-      <ul>
-      <li>5 Tribes (3 in moderated sessions)</li>
-      <li>6 States (3 in moderated sessions)</li>
-      </ul>
+<p>All together these STTs represent the following regions:</p>
+<ul>
+<li>Region 1 (2 STTs)</li>
+<li>Region 2 (1 STT)</li>
+<li>Region 3 (1 STT)</li>
+<li>Region 4 (2 STTs)</li>
```
Comment (ADPennington) [inline product-updates/index.html]:
```suggestion
<li>Region 4 (3 STTs)</li>
```

Comment (ADPennington) [reply product-updates/index.html]: @reitermb @lfrohlich @lilybl1

**Review by lfrohlich — APPROVED** [review]

**Review by ADPennington — APPROVED** [review]
thanks @reitermb @lilybl1! 

We'd like to revisit some language edits in the project updates. these can be handled in #2365

@andrew-jameson ready to merge.

**Pull request merged**